docs/backend/guides/how-to/create-a-custom-linter.md

```diff
@@ -0,0 +1,137 @@
+# How to Create a Custom Go Linter
+
+Creating custom GO linters can be a great way to analyze your 
+project's source code and alert you to bugs, errors, 
+or other issues with your code.
+
```

Comment:
It might be nice to add a table of contents, but I'm also fine if we don't.

Reply:
Agreed! And added.